packages/forma-36-react-components/src/components/Skeleton/SkeletonTable/SkeletonTableCell/SkeletonTableCell.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { Component } from 'react';
+
+import TableCell from '../../../Table/TableCell';
+import SkeletonContainer from '../../SkeletonContainer';
+import SkeletonBodyText from '../../SkeletonBodyText';
+
+export type SkeletonTableCellProps = {
+  clipId?: string;
+} & typeof defaultProps;
+
+const defaultProps = {
+  clipId: 'skeleton-table-cell',
+};
+
+export class SkeletonTableCell extends Component<SkeletonTableCellProps> {
+  static defaultProps = defaultProps;
+
+  render() {
+    const { clipId } = this.props;
+
+    return (
+      <TableCell>
+        <SkeletonContainer
+          svgHeight={42}
+          clipId={clipId}
+          gradientId={`${clipId}-gradient`}
+        >
+          <SkeletonBodyText numberOfLines={2} />
```

Comment:
Cool! It's also good because a single line is the default look of SkeletonBodyText :)

Thanks!

Reply:
![Screenshot 2020-03-17 at 09 44 34](https://user-images.githubusercontent.com/6597467/76838113-06ce6880-6834-11ea-8d0c-9266d5818aab.png)

Do you think we should centralize the line?

I don't want to decrease the cell height because this is the height that the cell will get with the data